### PR TITLE
[COLLECTIONS-757] Update the JavaDoc of the BidiMap

### PR DESCRIPTION
**Pull request by dota17**

### CODE REVIEW

**Comment by dota17** [comment]
Any one can help to solve the issue [](https://issues.apache.org/jira/browse/COLLECTIONS-759)?

**Comment by kinow** [comment]
> Any one can help to solve the issue [issue collections-759](https://issues.apache.org/jira/browse/COLLECTIONS-759)?

https://github.com/apache/commons-collections/pull/147 :+1:

**Comment by dota17** [comment]
Thanks for your PR @kinow . I have a little question why this issue didn't happen before ?

**Comment by kinow** [comment]
> Thanks for your PR @kinow . I have a little question why this issue didn't happen before ?

Don't mention it. Not sure @dota17. I hadn't noticed the build of `master` was broken until you linked that issue and I checked Travis on the project README in GitHub. Once that's fixed if you could rebase your branch here, Travis should build it with no issues :crossed_fingers:

File: src/main/java/org/apache/commons/collections4/bidimap/TreeBidiMap.java [hunk 1]
```diff
@@ -1287,32 +1287,32 @@ private static void checkKeyAndValue(final Object key, final Object value) {
     }
 
     /**
-     * increment the modification count -- used to check for
+     * Increment the modification count -- used to check for
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/collections4/bidimap/TreeBidiMap.java:1290]: Increment -> Increments

File: src/main/java/org/apache/commons/collections4/bidimap/TreeBidiMap.java [hunk 2]
```diff
@@ -1287,32 +1287,32 @@ private static void checkKeyAndValue(final Object key, final Object value) {
...
+     * Bumps up the size and note that the map has changed.
      */
     private void grow() {
         modify();
         nodeCount++;
     }
 
     /**
-     * decrement the size and note that the map has changed
+     * Decrement the size and note that the map has changed.
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/collections4/bidimap/TreeBidiMap.java:1307]: Decrement -> Decrements

**Comment by garydgregory** [comment]
@dota17 Please see my 2 comments

**Comment by dota17** [comment]
> @dota17 Please see my 2 comments

Thank you for your review @garydgregory . I will update it now.

**Pull request merged**